### DISCUSSION TITLE
[Q] Multi-instance, run job at most once

### DISCUSSION THREAD

**Discussion by bsdooby** [jobrunr/jobrunr#592] [Q&A] [answered]
Dear all, can JobRunr be used (just like ShedLock) in a multi instance environment to run a scheduled job exactly once (on node A); should node A be down, then run the job on Node B. But also only once?

**Comment by rdehuyss** [comment, marked as answer]
Yes

**Comment by bsdooby** [comment]
...and I think there are examples given in the documentation somewhere how to achieve this (?)

**Reply by nphotchkin** [reply]
See: [Make your background methods re-entrant](https://www.jobrunr.io/en/documentation/background-methods/best-practices/)

:)
